### PR TITLE
LGCS-770 - Create appropriate tests for a successful user login

### PR DESCRIPTION
**Pull request by joel-stobart**
https://digital.dclg.gov.uk/jira/browse/LGCS-770

### CODE REVIEW

**Comment by h0usemaintenance** [comment]
Thank you for creating the extra methods, they will come in handy for other tests. For acceptance/features/login.feature, the login will take the user to the assessment page after login rather than the index page, specific line is "Then I reach the index page" & "Then I reach the desired page"

**Review by h0usemaintenance — DISMISSED** [review]
As discussed elements in the tests will change with the frontend changes yet to be merged but this is a good base that can be easily refactored.

File: acceptance/helpers/FirefoxDriver.js [hunk 1]
```diff
@@ -28,37 +28,54 @@ class FirefoxDriver {
...
-            console.log("logging in", page_url);
             await this.driver.findElement(WebDriver.By.id('id_username')).sendKeys(this.username);
             await this.driver.findElement(WebDriver.By.id('id_password')).sendKeys(this.password);
             await this.driver.findElement(WebDriver.By.css('button')).click();
 
-
-            // await this.driver.wait(WebDriver.until.urlIs(url));
             let new_page_url = await this.driver.getCurrentUrl();
-
             console.log("Logged in to this page: ", new_page_url);
```
Comment (kingdomlevel) [inline acceptance/helpers/FirefoxDriver.js:53]: Can we get rid of this...?
```suggestion
```

**Review by kingdomlevel — CHANGES_REQUESTED** [review]
Would love if we can remove that console.log, otherwise happy to approve

**Pull request merged**